logging.c

```diff
@@ -180,13 +226,108 @@ static void owl_log_entry_free(void *data)
   }
 }
 
+#if GLIB_CHECK_VERSION(2, 32, 0)
```

Comment:
I think that a reasonable rule of thumb for version support[1] is "Do all supported versions of Ubuntu and Debian include a new enough version?", which this appears to satisfy: Wheezy has 2.32 (I think) (https://packages.debian.org/wheezy/libglib2.0-0) and Trusty has 2.40 (https://packages.ubuntu.com/trusty/libglib2.0-0). You don't need to remove the <2.32 support, but you can if you want to in a follow-on PR.

[1] I think we can be stricter -- I wouldn't be opposed to "dialups and the most recent Debian and Ubuntu LTS versions at least six months old", or something -- but this definitely seems fine. I guess I'm potentially open to an RHEL requirement too.